TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift

```diff
@@ -29,16 +29,18 @@ public protocol UpdateFactorPayload {
 public struct UpdatePushFactorPayload: UpdateFactorPayload {
   ///Factor Sid
   public let sid: String
-  ///Registration token generated by APNS when registering for remote notifications
-  public let pushToken: String
+  ///(Optional) Registration token generated by APNS when registering for remote notifications.
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/9cbd540c6135af3d1641ad17aef5ccc5e04d146c/TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift#L32"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>UpdateFactorPayload.swift:32</code></td>
    </tr>
  </tbody>
</table>